src/promptflow/promptflow/_sdk/operations/_local_storage_operations.py

```diff
@@ -400,6 +387,29 @@ def persist_node_run(self, run_info: NodeRunInfo) -> None:
         filename = f"{str(line_number).zfill(self.LINE_NUMBER_WIDTH)}.jsonl"
         node_run_record.dump(node_folder / filename, run_name=self._run.name)
 
+    def load_node_run_info(self, parse_const_as_str: bool = False) -> List[NodeRunInfo]:
+        json_loads = json.loads if not parse_const_as_str else json_loads_parse_const_as_str
+        node_runs = []
+        for node_folder in sorted(self._node_infos_folder.iterdir()):
+            for node_run_record_file in sorted(node_folder.iterdir()):
+                if node_run_record_file.suffix.lower() != ".jsonl":
+                    continue
+                with read_open(node_run_record_file) as f:
+                    new_runs = [json_loads(line)["run_info"] for line in list(f)]
```

Comment:
May handle KeyError in case the jsonl file not a valid run record.

Reply:
Let's add more error handling in the next PR, there're more error cases to be handled.